test/extensions/filters/http/fault/BUILD

```diff
@@ -54,6 +54,7 @@ envoy_extension_cc_test(
     name = "fault_filter_integration_test",
     srcs = ["fault_filter_integration_test.cc"],
     extension_names = ["envoy.filters.http.fault"],
+    shard_count = 2,
```

Comment:
nit: are these changes still necessary? Can they be reverted for now for clarity? Same below.